src/components/Cluster/NewCluster/ReleaseSelector/ReleaseRow.tsx

```diff
@@ -0,0 +1,138 @@
+import { ClassNames } from '@emotion/core';
+import styled from '@emotion/styled';
+import { relativeDate } from 'lib/helpers';
+import PropTypes from 'prop-types';
+import React, { FC, useMemo, useState } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+import { useSelector } from 'react-redux';
+import { getProvider } from 'selectors/mainInfoSelectors';
+import RadioInput from 'UI/Inputs/RadioInput';
+import ReleaseComponentLabel from 'UI/ReleaseComponentLabel';
+
+interface IReleaseRow extends IRelease {
+  isSelected: boolean;
+  selectRelease(releaseVersion: string): void;
+}
+
+const TableButton = styled(Button)`
+  height: 24px;
+  line-height: 24px;
+  position: relative;
+  top: -2px;
+  margin-left: 5px;
+  padding: 0px 15px;
+  i {
+    margin-right: 4px;
+  }
+`;
+
+const ComponentsWrapper = styled.div`
+  margin-left: ${({ theme }) => theme.spacingPx * 9}px;
+`;
+
+const Tr = styled.tr<{ isSelected: boolean }>`
+  background-color: ${({ isSelected, theme }) =>
+    isSelected ? theme.colors.foreground : 'transparent'};
+`;
+
+const StyledRadioInput = styled(RadioInput)`
+  margin-bottom: 0;
+`;
+
+const ReleaseRow: FC<IReleaseRow> = ({
+  components,
+  isSelected,
+  selectRelease,
+  timestamp,
+  version,
+}) => {
+  const provider = useSelector(getProvider);
+  const [collapsed, setCollapsed] = useState(true);
+  const kubernetesVersion = useMemo(
+    () =>
+      components.find((component) => component.name === 'kubernetes')?.version,
+    [components]
+  );
+
+  return (
+    <>
+      <Tr isSelected={isSelected} onClick={() => selectRelease(version)}>
+        <td>
+          <ClassNames>
+            {({ css }) => (
+              <StyledRadioInput
+                id={`select-${version}`}
+                title={`Select release v${version}`}
+                checked={isSelected}
+                value={isSelected ? 'true' : 'false'}
+                name={`select-${version}`}
+                onChange={() => selectRelease(version)}
+                bulletProps={{ className: css({ margin: 0 }) }}
```

Comment:
I wonder if we can declare this outside the component using the regular `css` function.

Example (with `css` imported from `emotion`:
```javascript
const BulletStyle = css`
	margin: 0;
`;
```

And then using it within `bulletProps` like (`css` from render props):
```javascript
...
bulletProps={{ className: css`${BulletStyle}` }}
...
```

That would really improve the readibility.

But would it work?

Reply:
I'll see if this works :+1:

Reply:
`You have tried to stringify object returned from `css` function. It isn't supposed to be used directly (e.g. as value of the `className` prop), but rather handed to emotion so it can handle it (e.g. as value of `css` prop).`

Reply:
At least we tried 😓